src/Adapter/Product/ProductUpdater.php

```diff
@@ -61,7 +62,7 @@ public function __construct(
      *
      * @throws CoreException
      */
-    public function update(Product $product, array $propertiesToUpdate, int $errorCode): void
+    public function update(Product $product, array $propertiesToUpdate, int $errorCode = 0): void
```

Comment:
Is the optional parameter necessary for Product, I though we had a dedicated error code for each sub domain so we might enforce that the error code should be provided in all cases?